javascript/src/components/extractors/MessageLoader.jsx

```diff
@@ -29,13 +34,23 @@ var MessageLoader = React.createClass({
 
         e.preventDefault();
     },
+    submit(messageId, index) {
+        this.refs.messageId.getDOMNode().value = messageId;
+        this.refs.index.getDOMNode().value = index;
+        this.refs.submitButton.getDOMNode().click();
```

Comment:
Please replace calls to `.getDOMNode()` with `React.findDOMNode()`, as that's the recommended way of doing it.